src/open_samus_returns_rando/lua_editor.py

```diff
@@ -213,6 +214,13 @@ def _create_custom_init(self, editor: PatcherEditor, configuration: dict) -> str
         # use _MAX if main is unlocked to unlock the ammo too
         if "ITEM_WEAPON_MISSILE_LAUNCHER" in inventory and "ITEM_MISSILE_TANKS" in inventory:
             inventory["ITEM_WEAPON_MISSILE_MAX"] = inventory.pop("ITEM_MISSILE_TANKS")
+        else:
+            # For the gun component to work without launcher, any value > 0 is sufficient
+            inventory["ITEM_MISSILE_CHECK"] = 1
+            samus_bmsad = editor.get_file(
+                "actors/characters/samus/charclasses/samus.bmsad", Bmsad
+            )
+            samus_bmsad.raw["components"]["GUN"]["functions"][20]["params"]["Param5"]["value"] = "ITEM_MISSILE_CHECK"
```

Comment:
Can't we just move the whole code back to where it was and always use `ITEM_MISSILE_CHECK` ?

Reply:
Hmm I guess I figured it'd be better to only have this be different if not shuffled. At one point I did check for the tunable value, but I removed that so there's no real benefit anymore. I can revert it back.